### PR TITLE
Bump to v5.1.0

### PR DESCRIPTION
**Pull request by mheon**
As the title says

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by ashley-cui** [comment]
LGTM

File: version/rawversion/version.go [hunk 1]
```diff
@@ -1,10 +1,10 @@
 package rawversion
-
+11;rgb:0000/0000/0000
```
Comment (ashley-cui) [inline version/rawversion/version.go:2]: Copy paste error?

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 ## 5.1.0
 ### Features
+- VMs created by `podman machine` on MacOS with Apple silicon can now use Rosetta 2 (a.k.a Rosetta) for high-speed emulation of x86 code. This is enabled by default. If you wish to change this option, you can use the `CONTAINERS_MACHINE_ROSETTA` environment variable or `containers.conf`.
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:5]:
```suggestion
- VMs created by `podman machine` on macOS with Apple silicon can now use Rosetta 2 (a.k.a Rosetta) for high-speed emulation of x86 code. This is enabled by default. If you wish to change this option, you can use the `CONTAINERS_MACHINE_ROSETTA` environment variable or `containers.conf`.
```

**Comment by TomSweeneyRedHat** [comment]
LGTM
I was just going to comment that you could ignore the macOS thing, but I'm glad you tidied it up.

**Comment by lsm5** [comment]
I guess the `typo in release notes` won't be going into the v5.1.0 tag ? No biggie I guess. LGTM

**Comment by mheon** [comment]
It'll still make it onto the GH release page so I think we're set

**Review by lsm5 — APPROVED** [review]
/lgtm
/approve
/hold

**Comment by mheon** [comment]
Looks like we're gonna pass
/hold cancel

**Pull request merged**